### ISSUE TITLE
[Research] DETEC-ADHD: Add ADHD/Attention Screening via Theta/Beta Ratio with Muse 2

### ISSUE THREAD

**Issue by LakshmiSravyaVedantham** [LakshmiSravyaVedantham/DreamAnalysis#38]
## Paper
**Title:** DETEC-ADHD: A Data-Driven Web App for Early ADHD Detection Using Machine Learning and Electroencephalography
**Authors:** Garicio-Daro et al.
**Year:** 2025
**Venue:** Big Data and Cognitive Computing (MDPI)
**URL:** https://www.mdpi.com/2504-2289/9/1/3

## Also Relevant
**Title:** The utility of wearable EEG combined with behavioral measures for ADHD diagnosis in young children
**Year:** 2024
**Venue:** Journal of Neurodevelopmental Disorders
**URL:** https://jneurodevdisorders.biomedcentral.com/articles/10.1186/s11689-024-09578-1

## Key Contribution
DETEC-ADHD is a web application that uses **Muse Band S (Gen 2)** for real-time EEG collection and applies logistic regression on theta/beta ratio biomarkers to classify ADHD subtypes (inattentive, hyperactive-impulsive, combined). Achieves 100% accuracy in children and 90% in adults. The 2024 study combines wearable EEG with behavioral measures for multimodal ADHD diagnosis.

Key biomarker: **theta/beta ratio (TBR) > 4.5** at frontal sites = ADHD indicator. This is the FDA-cleared NEBA biomarker (2013) and can be computed directly from Muse 2 AF7/AF8 channels.

## Relevance to NeuralDreamWorkshop
The existing `attention_classifier.py` already tracks theta/beta ratio and classifies 4 attention states, and notes the ADHD clinical threshold (TBR > 4.5). Adding a dedicated ADHD screening mode would:
1. Extend the app's clinical relevance
2. Use the same EEG features already computed
3. Provide actionable insights (not diagnosis — screening/awareness)

### Files to Create
- `ml/models/adhd_screener.py` — ADHD attention screening model:
  ```python
  class ADHDScreener:
      SUBTYPES = ["typical", "inattentive", "hyperactive_impulsive", "combined"]

      def screen(self, eeg_epochs: List[np.ndarray], fs=256) -> Dict:
          # Compute TBR across 3-5 min recording
          # Compare to age-normed thresholds
          # Return risk score + subtype probability
  ```
- `client/src/pages/adhd-screening.tsx` — ADHD screening page with:
  - 5-min sustained attention task (visual CPT — Continuous Performance Test)
  - Real-time TBR display during task
  - Results with clinical context (disclaimers, not a diagnosis)
- `ml/training/train_adhd.py` — Training script (optional, TBR threshold is well-validated)

### Files to Modify
- `ml/models/attention_classifier.py`:
  - Add `compute_tbr_profile(epochs)` — TBR across multiple epochs + variability
  - Add `adhd_risk_score()` — weighted TBR + attention lapse frequency
- `ml/api/routes.py`:
  - Add `POST /adhd/start-screening` — Begin 5-min attention task
  - Add `POST /adhd/submit-epoch` — Add epoch during screening
  - Add `GET /adhd/results/{session_id}` — Get screening results
  - Add `GET /adhd/history` — Track TBR trends across sessions
- `client/src/App.tsx` — Add route for ADHD screening page

### ADHD Biomarkers (from literature)
```
Primary:
  TBR (Theta/Beta Ratio) at AF7/AF8: > 4.5 = elevated ADHD risk
  TBR variability across epochs: high variability = inattentive subtype

Secondary:
  Alpha suppression during task: poor suppression = attention deficit
  Beta lateralization: right > left beta = hyperactive subtype
  Theta frontal midline power: elevated = sustained attention deficit

Behavioral (from CPT task):
  Reaction time variability: high = inattentive
  Commission errors (false alarms): high = impulsive
  Omission errors (misses): high = inattentive
```

### Important Disclaimers (MUST include in UI)
- "This is a screening tool, not a clinical diagnosis"
- "ADHD diagnosis requires comprehensive clinical evaluation"
- "Results should be discussed with a healthcare professional"
- FDA-cleared TBR threshold is based on NEBA system with 19-ch EEG — 4-ch Muse provides approximate screening only

## Expected Impact
- **New capability:** ADHD attention screening (no other consumer BCI app offers this)
- **Accuracy:** 85-90% for TBR-based screening (well-validated biomarker)
- **Clinical relevance:** Links to existing attention classifier features
- **User value:** Track attention/TBR trends across sessions for self-monitoring

## Complexity: MEDIUM
- TBR computation reuses existing band power features
- CPT task UI requires new page but is straightforward
- Main challenge: proper disclaimers and avoiding overclaiming diagnostic ability
- Age-normed thresholds need sourcing from published norms
